model/kubernetes/kubernetes_cluster.rb

```diff
@@ -0,0 +1,43 @@
+#  frozen_string_literal: true
+
+require_relative "../../model"
+
+class KubernetesCluster < Sequel::Model
+  one_to_one :strand, key: :id
+  one_to_one :api_server_lb, class: :LoadBalancer, key: :id, primary_key: :api_server_lb_id
+  many_to_one :private_subnet
+  many_to_many :vms do |ds|
+    ds.order_by(:created_at)
+  end
```

Comment:
nit: https://github.com/jeremyevans/sequel/blob/master/doc/association_basics.rdoc#label-3Aorder
```suggestion
  many_to_many :vms, order: :created_at
```